storm-client/src/jvm/org/apache/storm/executor/Executor.java

```diff
@@ -291,6 +294,22 @@ public void accept(Object event) {
         }
     }
 
+    public void setNeedToRefreshCreds() {
+        this.needToRefreshCreds = true;
+    }
+
+    protected void updateExecCredsIfRequired() {
+        if (this.needToRefreshCreds) {
+            this.needToRefreshCreds = false;
+            LOG.info("The credentials are being updated {}.", executorId);
```

Comment:
Maybe we should change it to be `DEBUG` since this is on a critical path (even though this `method` might not be called very frequently)

Reply:
I think info should be fine.